### PR TITLE
Add Form Feature in Html and Css

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Changes:

1- Add Form
2- Add Form Validations
3- Connecting Api to Form

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1067,13 +1075,105 @@ ul {
...
 }
 
+/* Form */
+
 .form-container {
-  border: 3px solid red;
+  border: 1px solid transparent;
   background: var(--main-purple-color);
   border-radius: 100px 0 0 0;
   background-image: url(assets/contact-form-shapes-mobile.svg);
```
Comment (Shisui6) [inline style.css:1084]: Kindly update the positioning of the background image in your contact form to better match the figma design🙏

Your design:
image

Figma design:
image

A few things to note:
- The shapes under the textarea are not showing

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Abdulwhbhussain and @WHB901, 

# Requires Changes 🍀
You have done a great job so far!👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🫡

## Highlights
- HTML form validation done correctly ✅
- Good design ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Abdulwhbhussain) [reply style.css:1084]: resolved the Issue.

**Review by vickymarz — APPROVED** [review]
Hi @Abdulwhbhussain, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Good UI design ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**